### PR TITLE
Refactor pageload metadata

### PR DESCRIPTION
**Pull request by rob-mccann**

### CODE REVIEW

File: autotracking/pageLoad.js [hunk 1]
```diff
@@ -1,38 +1,49 @@
...
+
+        if (name && content) {
+            data.meta['spt:' + name] = decodeURI(content);
+        } else if (isOGTag) {
+            data.og['spt:' + property] = decodeURI(content);
         }
     }
+
+    return data;
+}
```
Comment (magska) [inline autotracking/pageLoad.js]: autotracking/pageLoad.js: line 31, col 2, Missing semicolon.

Comment (rob-mccann) [inline autotracking/pageLoad.js]: :+1: forgot to push my change

File: autotracking/pageLoad.js [hunk 2]
```diff
@@ -1,38 +1,49 @@
...
-        if (ogFlag) {
-            pageload.addProperty('primary', 'spt:og', ogObject);
-        }
-        if (metaFlag) {
-            pageload.addProperty('primary', 'spt:meta', metaObject);
+        name    = metaTag.getAttribute('name');
+        content = metaTag.getAttribute('content');
+        property = metaTag.getAttribute('property');
+
+        isOGTag =  property && property.indexOf('og:') > -1;
```
Comment (magska) [inline autotracking/pageLoad.js]: Illegal trailing whitespace at ./autotracking/pageLoad.js :
    19 |        property = metaTag.getAttribute('property');
    20 |
    21 |        isOGTag =  property && property.indexOf('og:') > -1; 
---------------------------------------------------------------------^
    22 |
    23 |        if (name && content) {

I will look in to why this is not discovered by grunt watch.

Comment (rob-mccann) [inline autotracking/pageLoad.js]: I wasn't aware of that task :) I'll run it now. Does jshint exit on first
error too?

**Pull request merged**